<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="le_Tper_zd">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_Nc_zd">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>54</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bn_Send">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>97</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bn_Exit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>97</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bn_Update">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>105</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_Q_zd">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bn_Exit</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
